texthero/representation.py

```diff
@@ -180,17 +88,18 @@ def count(
     --------
     >>> import texthero as hero
     >>> import pandas as pd
-    >>> s = pd.Series(["Sentence one", "Sentence two"]).pipe(hero.tokenize)
-    >>> hero.count(s)
-    0  Sentence    1
-       one         1
-    1  Sentence    1
-       two         1
-    dtype: Sparse[int64, 0]
-
+    >>> s = pd.Series(["Sentence one", "Sentence two"]).pipe(
```

Comment:
Looks a bit ugly and `>>> s = pd.Series(["Sentence one", "Sentence two"]).pipe(hero.tokenize)` is 71 characters (less than 75, equal if we consider the initial tab of 4 characters). Let's keep in in one line :)

Same discussion for the other functions

Reply:
done for all where it's possible